graph/patterns/change-status.md

```diff
@@ -0,0 +1,107 @@
+# Change Status & Action Request
+
+Microsoft Graph API Design Pattern
+
+_The change status pattern is an effective alternative to model the ability to trigger side effects while keeping an agnostic, data driven approach_
+
+## Problem
+
+In some cases, exposed resources can trigger side effects that modify the system behavior. For instance, a [`riskyUser`](https://docs.microsoft.com/en-us/graph/api/resources/riskyuser?view=graph-rest-1.0) can be [`confirmedCompromised`](https://docs.microsoft.com/en-us/graph/api/riskyuser-confirmcompromised?view=graph-rest-1.0&tabs=http) or [`dismissed`](https://docs.microsoft.com/en-us/graph/api/riskyuser-confirmcompromised?view=graph-rest-1.0&tabs=http).
+
+These situation are often modeled as a `POST` request to a specific endpoint in the same URI space of the resource, triggering the side effect directly and not returning any data. While it might be a quick and easy solution, there are some considerations:
+
+- The API should be exposing resources and allow their manipulation, while Actions resemble a remote procedure call on an object, typical of the [OOP][1]
+- It couples the request with the side effect, making it more difficult to test, log, replay and reason about what is happening in the system
+- It forces the consumer to follow the [OOP][1] approach at the API level. We do not want to pursue any specific style and the developers writing the client might be working in other ways
+- A `POST` request is used to create a resource record. Using it to do other operations through an action is confusing, and should really be a last resort when there are no clear alternatives.
+
+## Solution
+
+API Designers can prefer the usage of a property indicating the status and request for its modification or a datafyed version of the action that represents the **intent** of triggering a side effect rather than starting the execution directly through the HTTP request.
+
+Taking in consideration the example of the `riskyUser` above, it is possible to keep the `riskState` property on the resource:
+
+```xml
+<EnumType Name="riskState">
+    <Member Name="confirmedCompromised" Value="0"/>
+    <Member Name="dismissed" Value="1"/>
+    <Member Name="none" Value="2"/>
+</EnumType>
+
+<EntityType Name="riskyUser" >
+    <Property Name="riskState" Type="riskState" />
+</EntityType>
+```
+
+To confirm the user as compromised, we can send a `PATCH` request on the resource:
+
+```bash
+curl -X PATCH --json '{"riskState": "confirmedCompromised"}' https://graph.microsoft.com/v1.0/identityProtection/riskyUsers/a57dc75f-24b5-47ce-b5e1-44822f5d4729
```

Comment:
could you please use  request examples without reference to any tool as in other patterns?